src/nodeparser.js

```diff
@@ -764,6 +764,10 @@ function renderableNode(node) {
 function isPositionedForStacking(container) {
     var position = container.css("position");
     var zIndex = (["absolute", "relative", "fixed"].indexOf(position) !== -1) ? container.css("zIndex") : "auto";
+    // In chrome fixed has a default zIndex of 0 instead of auto
+    if(parseFloat(zIndex, 10) === 0) {
```

Comment:
`parseFloat` doesn't have a second argument like `parseInt` has, and I believe `parseInt` would be more suitable here anyway.